reference/waterline/models/find.md

```diff
@@ -36,6 +38,39 @@ User.find({name:'Finn'}).exec(function (err, usersNamedFinn){
 ```
 
 
+##### Using projection
+
+Projection limits the fields returned on found records. This can be done, for example, for faster performance or greater security when passing found records to the client. The select argument takes an array of strings that correspond with record object keys. The record ID is always returned.
```

Comment:
Also, would you mind updating the last sentence in this paragraph along these lines?

> The `select` ~~argument~~clause in a [Waterline criteria](http://sailsjs.com/documentation/concepts/models-and-orm/query-language) takes an array of strings that correspond with ~~record object keys~~attribute names.

_re: "clause":_ We've been trying to formalize what we call the various segments of a criteria like `where`, `select`, etc. so it's consistent everywhere

_re: added link:_ This is just so it's easy to quickly jump to the complete documentation on WL criteria

_re: record object keys:_  There's nothing wrong with talking about the "keys of a record" -- and it's precise as well (by the time userland code gets a record, its keys are always attribute names, never column names).  The only reason I suggest a change here is to try and avoid introducing additional terminology.

Sorry to be annoying (and there's no way you could have known this since we haven't documented it anywhere yet!)